Tests/src/Main.cpp

```diff
@@ -32,7 +32,9 @@ int main(int argc, char* argv[])
 	Listeners.Append(Listener);
 #endif
 
-	LoadTestAccountCredentials();
+#if RUN_ALL_UNIT_TESTS || RUN_ORGANIZATIONSYSTEM_TESTS
+	LoadTestAccountCredentials(); // Needed as long as we cannot create superuser credentials on the fly
+#endif
```

Comment:
[Comment] Good idea conditionally compiling this, at least the credentials file can be ignored _mostly_